emsize.py

```diff
@@ -73,9 +88,10 @@ def print_sizes(js_file):
   for line in lines[:-1]:
     print(line)
 
-  print('JS\t\t%s\t0' % js_size)
-  print('Total\t\t%s' % total)
+  print('%s%16d%7d' % ('JS', js_size, 0))
+  print('%s%13s' % ('Total', total))
+  print('\n')
 
 
 if __name__ == '__main__':
-  sys.exit(print_sizes(parse_args(sys.argv[1:])))
+  sys.exit(parse_args(sys.argv[1:]))
```

Comment:
It seems odd to cal the main function here `parse_args`.    I recommend call this function `main` and then running `parse_args()` from there.. and then calling `print_sizes` after `parse_args`